src/Php73/bootstrap.php

```diff
@@ -9,8 +9,14 @@
  * file that was distributed with this source code.
  */
 
+use Symfony\Polyfill\Php73 as p;
+
 if (PHP_VERSION_ID < 70300) {
     if (!function_exists('is_countable')) {
-        function is_countable($var) { return is_array($var) || $var instanceof Countable || $var instanceof ResourceBundle || $var instanceof SimpleXmlElement; }
+        function is_countable($var) { return p\Php73::is_countable($var); }
+    }
+
+    if (!function_exists('hrtime')) {
+        function hrtime($as_num = false) { return p\Php73::hrtime($as_num); }
```

Comment:
Should be `$asNum`

Reply:
I wasn't sure if I should follow PSR or original function's parameter names here. I'll change it to PSR then.